app/javascript/src/Types/SwaggerTypes.ts

```diff
@@ -1,4 +1,18 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+
 export type AccountData = Record<string, {
   name: string;
   value: string;
 }[]>
+
+export interface ApiDocsService {
+  name: string;
+  system_name: string;
+  description: string;
+  path: string;
+  service_endpoint: string;
+}
```

Comment:
Missing blank line, for consistentcy
```suggestion
}

```